backends/magma/ceed-magma.c

```diff
@@ -630,7 +631,9 @@ static int CeedBasisApply_Magma(CeedBasis basis, CeedInt nelem,
   const CeedInt dim = basis->dim;
   const CeedInt ncomp = basis->ncomp;
   const CeedInt nqpt = ncomp*CeedIntPow(basis->Q1d, dim);
+  #ifndef USE_MAGMA_BATCH4
```

Comment:
Is this tested both ways?

Reply:
Yes, if USE_MAGMA_BATCH4 is not defined, a slower version of the backend will be called.